core/src/main/java/net/adoptopenjdk/icedteaweb/client/commandline/CommandLine.java

```diff
@@ -73,6 +74,11 @@
  */
 public class CommandLine {
 
+    static {
+        // this is placed here above the any thing else to ensure no logger has been created prior to this line
+        FileLog.setLogFileNamePostfix("settings");
```

Comment:
Why does the CommandLine has the settings prefix?

Reply:
the `main()` in CommandLine is used by ITW to start the settings UI.
This change marks the logs from running the settings by appending "settings" to the log file name